server/src/testFixtures/java/io/crate/analyze/TableDefinitions.java

```diff
@@ -107,12 +92,6 @@ public final class TableDefinitions {
         new PartitionName(new RelationName("doc", "parted"), singletonList(null)).asIndexName()
     };
 
-    public static final String TEST_EMPTY_PARTITIONED_TABLE_DEFINITION =
-        "create table doc.empty_parted (" +
-        "  name text," +
-        "  date timestamp with time zone" +
-        ")" +
-        " partitioned by (date)";
     public static final String PARTED_PKS_TABLE_DEFINITION =
```

Comment:
do I get it right, that we remove only those definitions, which are used only in 1 place but still keep tables with multi usage?

Reply:
yes